en/appendices/5-0-migration-guide.rst

```diff
@@ -49,6 +56,11 @@ Controller
   change can impact applications that use ``property_exists()`` on components.
 - The components' ``Controller.shutdown`` event callback has been renamed from
   ``shutdown`` to ``afterFilter`` to match the controller one. This makes the callbacks more consistent.
+- ``PaginatorComponent`` has been removed and should be replaced by using
```

Comment:
In controller's one can just use paginate(), no need to use the paginator directly. Also the class is now NumericPaginator.